pdf_parser/objects/PdfObjects.py

```diff
@@ -115,6 +112,7 @@ def get_lines_by_font_size(self, font_size):
         lines_results = []
         for page in self.pages:
             lines = [line for line in page.lines if line.size == font_size]
+            lines = list(filter(lambda x: x.size == font_size, page.lines))
```

Comment:
although i have to say that guido and the python community is divided on the subject on what is more pythonic